cinema/serializers.py

```diff
@@ -34,6 +43,19 @@ class MovieListSerializer(MovieSerializer):
     actors = serializers.SlugRelatedField(
         many=True, read_only=True, slug_field="full_name"
     )
+    tickets_taken = serializers.IntegerField(read_only=True, source="tickets.count")
```

Comment:
The `tickets_taken` field correctly uses `source="tickets.count"` to count the number of tickets related to a movie. This is a good use of the `source` attribute to derive a value from a related field.